resources/quality/fusion3/fusion3_0.4_ABS_fine.inst.cfg

```diff
@@ -0,0 +1,42 @@
+[general]
+version = 4
+name = Fine Quality
+definition = fusion3
+
+[metadata]
+setting_version = 19
+type = quality
+quality_type = fine
+material = generic_abs
+variant = 0.4mm Nozzle
+
+[values]
+layer_height = 0.1
+layer_height_0 = 0.25
+material_bed_temperature = 110
+retraction_combing = noskin
+cool_fan_enabled = True
+cool_fan_full_layer = 7
+cool_fan_speed = 65
+cool_fan_speed_min = 15
+cool_min_layer_time = 3
+cool_min_layer_time_fan_speed_max = 6
+
+infill_sparse_density = 75
+inset_direction = outside_in
+line_width = 0.4
+adhesion_type = brim
+material_flow = 98
+material_print_temperature = 255.0
```

Comment:
In a way, yes thats exactly what the code is doing, however following the method you have posted, ends up causing a problem. The temperatures listed in the profiles I put in are how the original profiles are setup by Fusion3. 

I am currently reworking the entire PR in order to make it compatible with 5.1.1. I have closed this PR and will be re-pulling my repository because there seems to be some kind of fault that will not stop popping up in my email.

If the PR isnt accepted once its reposted, I'll just keep it in patch form over on the Fusion360 group. They have all seemed to like the 4.13.1 patch very much.